subsys/spm/Kconfig

```diff
@@ -215,7 +215,7 @@ endmenu
 
 if ARM_FIRMWARE_USES_SECURE_ENTRY_FUNCS
 config ARM_ENTRY_VENEERS_LIB_NAME
-	string
+	string "Secure entry function library"
```

Comment:
We'll need a prompt-guard for this to protect calculation of default values.

https://github.com/zephyrproject-rtos/zephyr/pull/20702/files

Reply:
exactly what I was looking for, thanks

Reply:
Fixed